src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/Transaction.java

```diff
@@ -0,0 +1,45 @@
+package uk.nhs.digital.nhsconnect.nhais.model.edifact;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+public class Transaction extends Section {
```

Comment:
Is the idea to?
1) subclass Transaction for specific transaction types; or
2) have accessors for all possible segments and let the mappers "pick" what they need

Reply:
(2)